index.js

```diff
@@ -57,10 +57,13 @@ module.exports = (config) => {
         return Date.now() - startTime
       },
 
-      tick: function(stat, num, _sampleRate, _tags, distribution=false) {
+      tick: function(stat, num, _sampleRate, _tags, options) {
         const count = num || 1;
         const sampleRate = _sampleRate || 1;
         const tags = _tags || null;
+        const defaultOptions = { distribution:false, increment:false, timing:true };
```

Comment:
I think that `increment:false`  is the breaking change.
Is this as intended?

Reply:
That's true.
I will fix it.

Reply:
For example, how about making this defaultOptions configurable when init  like `config.tickDefaultOptions` ?